packages/flutter/lib/src/material/text_field.dart

```diff
@@ -123,7 +123,10 @@ class TextField extends StatefulWidget {
   /// characters may be entered, and the error counter and divider will
   /// switch to the [decoration.errorStyle] when the limit is exceeded.
   ///
-  /// The [textAlign], [autofocus], [obscureText], [autocorrect],
+  /// If [showCursor] is not set or is null, it will be set to false if [readOnly] is
```

Comment:
This is a little confusing since it refers to "setting" the property and it's hard not to read it as meaning that showCursor is always false. Maybe: 

The text cursor is not shown if [showCursor] is false or if [showCursor] is null (the default) and [readOnly] is true.